dashboard/test/controllers/script_levels_controller_test.rb

```diff
@@ -1487,4 +1487,15 @@ def get_hidden(script)
 
     assert_equal 1, SectionHiddenStage.where(section_id: section.id).length
   end
+
+  test "should redirect when script has a redirect_to property" do
+    script = create :script
+    new_script = create :script
+    create(:script_level, script: script)
+    create(:script_level, script: new_script)
+    script.update(redirect_to: new_script.name)
+
+    get :show, params: {script_id: script.name, stage_position: '1', id: '1'}
+    assert_redirected_to "/s/#{new_script.name}/stage/1/puzzle/1"
```

Comment:
Still testing redirect from `a/_/1/_/1` to `b/_/1/_/1`, but the more I consider this the more I think it's a non-issue; we never really settled on 'spec' for where to go in the new script, so probably not necessary to pin it.